src/utils.jl

```diff
@@ -532,6 +532,94 @@ julia> zeros_like(x, Float64)
 zeros_like(x::AbstractArray, T::Type, sz=size(x)) = fill!(similar(x, T, sz), 0)
 zeros_like(x::AbstractArray, sz=size(x)) = zeros_like(x, eltype(x), sz)
 
+"""
+    rand_like([rng=GLOBAL_RNG], x, [element_type=eltype(x)], [dims=size(x)])
```

Comment:
Minor thing but in Julia 1.7+ this is `default_rng()` which is different from `GLOBAL_RNG`

Reply:
@theabhirath can you file a new or for this? We can add a method returning the default rng conditional on the Julia version

Reply:
Here's the thing, the current method doesn't actually have `rng=GLOBAL_RNG` as a default since the default RNG handling is done all by `rand!` or `randn!`. So the change is limited to the docs, I suppose? I'd file a PR for that but not sure if docs should refer to LTS or 1.7+

Reply:
Yeah the change is limited to the docs which is why I didn't think it should block the PR. I would let it reflect the latest Julia version.

Reply:
Oh wait. This is funny, but the docs for 1.7.3 themselves used `GLOBAL_RNG` (that's where I copied it from, now that I think of it) https://docs.julialang.org/en/v1/stdlib/Random/#Base.randn. Not sure what's going on here